doc/concepts/configuration/configuration_etcd.rst

```diff
@@ -6,92 +6,150 @@ Storing configuration in etcd
 
-    .. code-block:: yaml
+To publish a cluster's configuration (``cluster.yaml``) to an etcd server, execute ``tt cluster publish`` as follows:
 
-        config:
-          etcd:
-            http:
-              request:
-                timeout: 3
-            prefix: /tt
-            endpoints:
-            - http://localhost:2379
-            username: testuser
-            password: foobar
+.. code-block:: console
 
+    $ tt cluster publish "http://localhost:2379/example" instances.enabled/app/cluster.yaml
+
+.. NOTE::
+
+    You can see a cluster's configuration using the :ref:`tt cluster show <tt-cluster>` command.
+
+
+.. _etcd_publishing_configuration_etcdctl:
+
+Publishing configuration using etcdctl
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To publish a cluster's configuration using the ``etcdctl`` utility, use the ``put`` command:
+
+.. code-block:: console
+
+    $ etcdctl put /example/config/all < cluster.yaml
+
+
+
+
+.. _etcd_starting_instances:
+
+Starting Tarantool instances
+----------------------------
+
+To learn how to start Tarantool instances, see the :ref:`Starting Tarantool instances <configuration_run_instance>` section.
+
+
+.. _etcd_reloading_configuration:
+
+Reloading configuration
+-----------------------
+
+By default, Tarantool watches etcd keys with the :ref:`specified prefix <etcd_local_configuration>` for changes in a cluster's configuration and reloads a changed configuration automatically.
+If necessary, you can set the :ref:`config.reload <configuration_reference_config_reload>` option to ``manual`` to turn off configuration reloading:
+
+.. code-block:: yaml
+
+    config:
+      reload: 'manual'
+      etcd:
+        # ...
+
+In this case, you can reload a configuration in the :ref:`application code <configuration_application>` using the ``reload()`` function provided by the :ref:`config <config-module>` module:
+
+.. code-block:: lua
+
+    require('config'):reload()
+
+
+
+
+
+
+
+
+
+..
+    Generating certificates for testing:
+    1) openssl genrsa -out ca.key 2048
+    2) openssl req -new -x509 -days 365 -key ca.key -subj "/C=CN/ST=GD/L=SZ/O=Acme, Inc./CN=Acme Root CA" -out ca.cr
+    3) openssl req -newkey rsa:2048 -nodes -keyout server.key -subj "/C=CN/ST=GD/L=SZ/O=Acme, Inc./CN=localhost" -out server.csr
+    4) openssl x509 -req -extfile <(printf "subjectAltName=DNS:localhost,IP:127.0.0.1") -days 365 -in server.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out server.crt
+    5) sudo cp server.crt /etc/ssl/certs
+    6) sudo cp server.key /etc/ssl/private
+
+    Starting etcd:
+    etcd --cert-file=ssl/server.crt --key-file=ssl/server.key --advertise-client-urls=https://localhost:2379 --listen-client-urls=https://localhost:2379
+
+    Get keys:
+    etcdctl get /tt/config/all --cert=ssl/server.crt --key=ssl/server.key
```

Comment:
It gives just `<prefix>/config/all`, not `<prefix>/config/*`. I suggest `etcdctl get /tt/config/ /tt/config0` (it actually means `/tt/config/*`, because `0` is the next character after `/`).

Reply:
Thanks for hints. Note that `Generating certificates for testing` section is commented and not rendered in docs. I just copied/pasted the commands I used not to forget them in the future :)